pyccel/codegen/printing/ccode.py

```diff
@@ -714,6 +714,8 @@ def _print_PythonMin(self, expr):
             return "fmin({}, {})".format(self._print(arg[0]),
                                          self._print(arg[1]))
         elif arg.dtype.primitive_type is PrimitiveIntegerType() and len(arg) == 2:
+            if isinstance(arg[0], Variable) and isinstance(arg[1], Variable):
+                return f"({self._print(arg[0])} < {self._print(arg[1])} ? {self._print(arg[0])} : {self._print(arg[1])})"
             arg1 = self.scope.get_temporary_variable(PythonNativeInt())
             arg2 = self.scope.get_temporary_variable(PythonNativeInt())
```

Comment:
Can the extra argument be avoided for each argument individually?

Currently the following case will also create an unnecessary variable:
```python
r = max(a, b+c)
```